### DISCUSSION TITLE
(newbie) checkbox custom color

### DISCUSSION THREAD

**Discussion by mheepandaaa** [saadeghi/daisyui#2359] [Q&A] [answered]
How can I customize the checkbox color like the radio bg color?
![image](https://github.com/saadeghi/daisyui/assets/72680344/050c88c5-7fe0-4200-a0b0-1887446a767e)

**Comment by saadeghi** [comment, marked as answer]
For `radio`: the last example on this page is using custom colors:  
https://daisyui.com/components/radio/#with-label-and-form-control-and-custom-colors
```html
<input type="radio" class="radio checked:bg-red-500" checked />
```

For `checkbox` you can either change the CSS variables like this: https://play.tailwindcss.com/MwBJTMq0JG?file=css
Or use Tailwind class names like this (numbers are HSL values)
```html
<input
  type="checkbox"
  class="checkbox [--chkbg:0_0%_0%] [--chkfg:50_100%_50%]"
>
```
